tests/test_dump.py

```diff
@@ -47,7 +75,7 @@ class TestSchema(Schema):
         json_schema = JSONSchema()
         dumped = json_schema.dump(schema).data
         self._validate_schema(dumped)
-        assert dumped['properties']['myfield']['title'] == 'Brown Cowzz'
+        dumped['properties']['myfield']['title'] == 'Brown Cowzz'
```

Comment:
Whoops sometimes I get careless with my vim. I'll toss it back in there.
On Sun, Jul 10, 2016 at 13:35 Stephen J. Fuhry <notifications@github.com>
wrote:

> In tests/test_dump.py
> <https://github.com/fuhrysteve/marshmallow-jsonschema/pull/14#discussion_r70185531>
> :
>
> > @@ -47,7 +75,7 @@ class TestSchema(Schema):
> >          json_schema = JSONSchema()
> >          dumped = json_schema.dump(schema).data
> >          self._validate_schema(dumped)
> > -        assert dumped['properties']['myfield']['title'] == 'Brown Cowzz'
> > +        dumped['properties']['myfield']['title'] == 'Brown Cowzz'
>
> Is there a reason for dropping the assert here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fuhrysteve/marshmallow-jsonschema/pull/14/files/15d912509dc20ce9f1277f9e72d53ef75507156b#r70185531>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAa2gMrp_iSQp0tjjNR1MjHMeR5CsNTzks5qUS1SgaJpZM4JIRfW>
> .
>
-- 
Eli Gundry
330.828.6147